MetaMask/metamask-extension
# Replacing deprecated constants and adding new stories

## Explanation

- This PR partially fixes following issues
  - #18714 
  - #19526 
  - #17670 
- Replaced deprecated constants and created stories for `SnapRemoveWarning` and `SnapUIMarkdown`

1. `ui/components/app/snaps/snap-remove-warning`
2. `ui/components/app/snaps/snap-ui-markdown`

## Screenshots/Screencaps


### `SnapRemoveWarning`
<img width="1440" alt="Screenshot 2023-06-13 at 7 44 14 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/0a6f12e3-7aa8-407c-ba78-35226fdeec83">

### `SnapUIMarkdown`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/d3c431a2-1a88-41de-b5b8-ce4664d642c3)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/f512ea61-5c74-4f7b-8664-70788951ae38)


## Manual Testing Steps

1. Pull this branch 
2. Run `yarn storybook`
3. Check `SnapRemoveWarning` and `SnapUIMarkdown`



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
